src/modules/ProductLicense.rb

```diff
@@ -778,6 +785,44 @@ def display_beta(id)
       end
     end
 
+    # Determines whether the license was accepted for a given product
+    #
+    # This method reads the license content and uses the new
+    # Y2Packager::ProductLicense API to determine whether the license was
+    # accepted or not. This is just a transitory method because, in the future,
+    # all licenses should be read through the new API.
+    #
+    # @param product  [Y2Packager::Product] Product instance
+    # @param licenses [Hash<String,String>] License files indexed by language
+    # @return [boolean] true if the license was already accepted; false if it was
+    #   not acceptedd or it is not found.
+    def license_accepted_for?(product, licenses)
+      license_file = licenses["en_US"] || licenses["en"] || licenses[""]
+      content = SCR.Read(path(".target.string"), license_file).to_s
+      if content.empty?
+        log.info "No license was found for #{product.name} in the repository"
```

Comment:
are you sure it is just info? and do you test how it behave for this case?

Reply:
Unit tests should cover this case. I will raise it to error.